src/__test__/Nearby.test.js

```diff
@@ -27,7 +29,7 @@ describe('Testing Nearby component', () => {
     let wrapper = shallow(<OgNearby {...prop} />);
 
     // mocking fetch
-    fetch = jest.fn((arg) => Promise.resolve({
+    global.fetch = jest.fn((arg) => Promise.resolve({
```

Comment:
is this necessary? or can we use another declared variable in this scope and reuse. I've not checked if this is being reused.